w_droplet_clustering/scanpy_clustering_workflow_parameters.yaml

```diff
@@ -38,128 +36,216 @@ filter_genes:
         subset:
             __class__: ConnectedValue
 find_clusters:
-    __job_resource:
-        __current_case__: 0
-        __job_resource__select: 'no'
     input_format: anndata
     method: louvain
     output_cluster: 'true'
-    output_format: anndata
+    output_format: anndata_h5ad
     settings:
         __current_case__: 1
         default: 'false'
         directed: 'true'
-        key_added: ''
+        key_added: METHOD_resolution_RESOLUTION
+        layer: ''
+        neighbors_key: neighbors
         random_seed: '1234'
         resolution: '1.0'
         resolution_file:
-            __class__: ConnectedValue
+            __class__: RuntimeValue
         restrict_to: ''
-        use_graph: neighbors
         use_weights: 'false'
 find_markers:
-    groupby: louvain
+    groupby: INPUT_OBJ
```

Comment:
This INPUT_OBJ is declared above and do you plan to replace this wildcard in the parameter file? I'm trying to understand the usage of this var

Reply:
Yes, this is a dirty hack that allows the marker script to 'know' the slot where the upstream clustering step stored its cluster definitions, and that information is passed via the input object file name. INPUT_OBJ is a wildcard that will be used internally. I've intentionally not documented it as it is a hack, and it will be replaced with more comprehensive analysis metadata handling in future.